src/Multipart/UploadState.php

```diff
@@ -25,12 +37,15 @@ class UploadState
     /** @var int Identifies the status the upload. */
     private $status = self::CREATED;
 
+    private $progressThresholds = [];
+
     /**
      * @param array $id Params used to identity the upload.
      */
     public function __construct(array $id)
     {
         $this->id = $id;
+        echo array_shift($this->progressBar);
```

Comment:
This should be moved to be a part of the display progress method with a threshold of 0